.github/workflows/build.yml

```
@@ -56,7 +56,7 @@ jobs:
           bun run vscode:prepublish
 
       - name: 🏷️ Manage label based on build result
-        if: ${{ always() }}
+        if: ${{ github.event_name == 'pull_request' }}
```

Don't we need the "always" here?

```suggestion
        if: ${{ always() && github.event_name == 'pull_request' }}
```

---

No